lib/websocket.js

```diff
@@ -104,12 +104,17 @@ function init (env, ctx, server) {
     }
   }
 
+  function pingClients() {
+    io.to('PingReceivers').emit('ping');
+  }
+  
   function listeners ( ) {
     io.sockets.on('connection', function (socket) {
       var socketAuthorization = null;
       var clientType = null;
       var timeDiff;
       var history;
+      var pingme;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ["pingme" is defined but never used](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/2093601353/issues/source?bid=2452379&fileBranchId=3141476#l117)